<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenShotForm</class>
 <widget class="QWidget" name="ScreenShotForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>239</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="screenshotMenu">
    <item>
     <widget class="QPushButton" name="btn_savefile">
      <property name="styleSheet">
       <string notr="true">QPushButton#btn_savefile
{
	background-color:rgba(0,0,0,0);
	color:rgb(250,161,65);
}
QPushButton#btn_savefile:hover
{
	background-color:rgb(100,250,100);
	color:rgb(250,201,110);
}

QPushButton#btn_savefile:pressed
{
	background-color:rgb(100,250,100);
	color:rgb(250,248,107);
}</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_saveclip">
      <property name="styleSheet">
       <string notr="true">QPushButton#btn_saveclip
{
	background-color:rgba(0,0,0,0);
	color:rgb(250,161,65);
}

QPushButton#btn_saveclip:hover
{
	background-color:rgb(250,100,100);
	color:rgb(250,201,110);
}

QPushButton#btn_saveclip:pressed
{
	background-color:rgb(250,100,100);
	color:rgb(250,248,107);
}</string>
      </property>
      <property name="text">
       <string>完成</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_closeRect">
      <property name="styleSheet">
       <string notr="true">QPushButton#btn_closeRect
{
	background-color:rgba(0,0,0,0);
	color:rgb(250,161,65);
}
QPushButton#btn_closeRect:hover
{
	background-color:rgb(100,100,250);
	color:rgb(250,201,110);
}

QPushButton#btn_closeRect:pressed
{
	background-color:rgb(100,100,250);
	color:rgb(250,248,107);
}</string>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_size">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>22</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(150, 150, 150);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="screenscaled">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
